ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -17,7 +17,9 @@
 import com.github.ambry.clustermap.ClusterMap;
 import com.github.ambry.clustermap.ClusterMapUtils;
 import com.github.ambry.clustermap.DataNodeId;
+import com.github.ambry.clustermap.HardwareState;
```

Comment:
are there specific test cases for `Disk_Unavailable` v/s `Replica_Unavailable`?

Reply:
line193 ~ line204 is testing for `Replica_Unavailable` and `Disk_Unavailable` respectively.
Another example is line457 ~ line 466. The testings are embedded in different types of admin requests testings.